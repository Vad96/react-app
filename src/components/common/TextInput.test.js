import React from 'react';
import TextInput from './TextInput';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<TextInput onChange={jest.fn()} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
