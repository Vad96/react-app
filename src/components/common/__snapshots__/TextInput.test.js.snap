// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form-group"
>
  <label />
  <div
    className="field"
  >
    <input
      className="form-control"
      onChange={[MockFunction]}
      type="text"
    />
  </div>
</div>
`;
